cli/src/cli_util.rs

```diff
@@ -3065,7 +3068,7 @@ impl CliRunner {
         }
 
         let settings = UserSettings::from_config(config);
-        let command_helper = CommandHelper {
+        let command_helper = Rc::new(CommandHelper {
```

Comment:
I think it's better to wrap inner data by `Rc` (something like `CommandHelper { data: Rc<CommandHelperData> }`) than changing function signature.